metricbeat/modules.d/kibana-xpack.yml.disabled

```diff
@@ -0,0 +1,12 @@
+# Module: kibana
```

Comment:
I'm a bit surprised that the collection of these files just works. Or did you move these manually?

Reply:
Me too, TBH! However, it works; I just re-tested it twice to make sure, like so:

```
rm -rf modules.d/
mage update
ls -al modules.d/ | grep xpack
```

Do you mind pulling down this PR and trying this out yourself as well, just in case?